src/main/java/seedu/address/model/ingredient/UniqueIngredientList.java

```diff
@@ -46,7 +46,7 @@ public void add(Ingredient toAdd) {
         internalList.add(toAdd);
     }
 
-    public void setIngredients(UniqueIngredientList replacement) {
+    public void setIngredient(UniqueIngredientList replacement) {
```

Comment:
Based on the existing UniquePersonList, there should be setIngredients(UniqueIngredientList), and setIngredient(Ingredient) which was missing. I fixed this in my PR as well.

Reply:
I currently have ```setIngredient(uniqueIngredientList)``` and ```setIngredients(List<Ingredient>)```. Are you suggesting to add a ```setIngredient(Ingredient)```?

Reply:
yep, that's based on the uniquePersonList, we should have
- `setIngredient(Ingredient)`
- `setIngredients(List<Ingredient>)`
- `setIngredients(UniqueIngredientList)`
which we all need i think.

Reply:
i guess i can fetch and merge with your version